tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,157 @@
+"""
+.. _point_spread:
+
+Corrupt known signal with point spread
+======================================
+
+The aim of this tutorial is to demonstrate how to put a known signal at a
+desired location(s) in a :class:`SourceEstimate` and then corrupt the signal
+with point-spread by applying a forward and inverse solution.
+"""
+
+import os.path as op
+
+import numpy as np
+
+import mne
+from mne.datasets import sample
+
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+from mne.simulation import simulate_stc, simulate_evoked
+
+###############################################################################
+# Parameters
+
+seed = 8675309
+
+# regularization parameter for inverse method
+method = 'sLORETA'
+snr = 3.
+lambda2 = 1.0 / snr ** 2
+
+# signal simulation parameters
+# do not add extra noise to our know signals
+noise_snr = np.inf
+T = 100
+times = np.linspace(0, 1, T)
+dt = times[1] - times[0]
+
+# Paths to MEG data
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects/')
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis-meg-oct-6-fwd.fif')
+fname_inv = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis-meg-oct-6-meg-fixed-inv.fif')
+
+fname_evoked = op.join(data_path, 'MEG', 'sample',
+                       'sample_audvis-ave.fif')
+
+subject_dir = op.join(data_path, 'subjects/')
+
+###############################################################################
+# Load and process MEG data
+fwd = mne.read_forward_solution(fname_fwd, force_fixed=True,
+                                surf_ori=True)
+inv_op = read_inverse_operator(fname_inv)
+
+# We need the epochs data to compute the covariance, so we manually compute the
+# evoked data by averaging the epochs.
+evoked_ff = mne.read_evokeds(fname_evoked, condition=0, baseline=(None, 0))
```

Comment:
why do you need to read this? you have it below if you average the epochs. Just set the tmin, tmax to the good values